t/api

```diff
@@ -949,6 +1019,64 @@ try "Pausing and unpausing jobs" # {{{
 
 # }}}
 
+
+try "Finding a single user" # {{{
+(run user $(uuidof user admin)
+ context 'a single user'
+ attr uuid    '355ccd3f-1d2f-49d5-937b-f4a12033a0ce'
+ attr name    'admin'
+ attr account 'admin'
+ attr sysrole 'admin'
+) 2>&1 | indent
+
+
+# }}}
+
+
+try "Listing users" # {{{
+(run users
+ context 'all users'
+ includes user admin
+ includes user exampleuser
+
+ pick exampleuser
+ attr uuid    '355ccd3f-1d2f-49d5-937b-f4a12033a0cf'
+ attr name    'exampleuser'
+ attr account 'exampleuser'
+ attr sysrole 'technician'
+
+ run users --fuzzy a
+ context 'users with account "*a*"'
+ includes user exampleuser
+ includes user admin
+
+ run users --fuzzy m
+ context 'users with account "*m*"'
+ includes user exampleuser
+ includes user admin
+
+ run users --fuzzy xyzzy
+ none 'no users with account "*xyzzy*"'
+
+ run users --sysrole admin
+ context 'users with sysrole admin'
+ excludes user exampleuser
+ includes user admin
+
+ run users --sysrole technician
+ context 'users with sysrole technician'
+ includes user exampleuser
+ excludes user admin
+
+ run users --sysrole manager
+ context 'users with sysrole manager'
+ none 'no users with sysrole "manager"'
+
```

Comment:
We have no --backend tests -- is `--backend` being honored?

Reply:
see above comment